.github/workflows/convert.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo "Converting all .md tutorials to .py files..."
+cd ./docs/tutorials;
```

Comment:
```suggestion
TUTS_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../../docs/tutorials"
pushd ${TUTS_DIR}
```

I made a suggestion on Slack that was wrong, but this one [appears to be reliable](https://stackoverflow.com/a/4774063).